### ISSUE TITLE
Blending issues

### ISSUE THREAD

**Issue by haseebyf** [asinesio/cocos2d-PRKit#3]
PRFilledPolygon has some blending issues. Try putting some background sprite in the example code.

**Comment by jnutting** [comment]
Yeah, different triangles seem to have different blending characteristics. The last one in the list seems to render OK, but the others blend with the background to varying degrees. I've found a workaround by changing the draw method to render each triangle one at a time. I don't know why the original fails but this works (I'm still too new at OpenGL to see understand what the difference is).

``` objective-c
-(void) draw {
    // we have a pointer to vertex points so enable client state
    glBindTexture(GL_TEXTURE_2D, self.texture.name);

    glTexEnvf(GL_TEXTURE_ENV, GL_TEXTURE_ENV_MODE, GL_DECAL);

    for (int i = 0; i <= areaTrianglePointCount; i+= 3) {
        glVertexPointer(2, GL_FLOAT, 0, areaTrianglePoints + i);
        glTexCoordPointer(2, GL_FLOAT, 0, textureCoordinates + i);

        glDrawArrays(GL_TRIANGLES, 0, 3);
    }
    glTexEnvf(GL_TEXTURE_ENV, GL_TEXTURE_ENV_MODE, GL_MODULATE);

    //Restore texture matrix and switch back to modelview matrix
    glEnableClientState(GL_COLOR_ARRAY);
}
```

**Comment by asinesio** [comment]
Calling `glDrawArrays()` per triangle like this is going to kill your performance, so I don't recommend it.   

Please see if the issue is fixed for you under cocos2d 2.0 (and the latest PRKit.)  I'll take a look when I have some time.
